zgrab_schema.py

```diff
@@ -205,7 +205,11 @@
         "value":Binary(),
         "length":Integer(),
         "lifetime_hint":Long()
-    })
+    }),
+    "crypto_variables":SubRecord({
```

Comment:
I'm not convinced this is the most descriptive thing to call this, or place to put these variables. Practically everything in the TLS handshake is a crypto variable. pre-master-secret should go under client_key_exchange. Not totally sure where we want to put master_secret.